projects/admin/src/app/record/detail-view/permission-detail-view/permission-detail-view.component.html

```diff
@@ -0,0 +1,53 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2022 RERO
+  Copyright (C) 2022 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="container" *ngIf="permissionsByRoles">
+  <h1 translate>Permissions matrix</h1>
+    <div class="my-3">
+      <label for="filter" translate>Filter permission list</label>
+      <input id="filter" class="form-control" #search (keyup)="filterPermissions(search.value)" />
+      <small id="filterInfo" class="form-text text-muted">Enter a part of the permission name to filter the list.</small>
+    </div>
+    <ng-container *ngIf="filteredPermissions.length >0; else noResult">
+      <table class="table table-header-rotated table-striped">
+        <thead>
+          <tr>
+            <th class="vertical-header">&nbsp;</th>
+            <th class="vertical-header" *ngFor="let role of roles">
+              <span>{{ role | translate }}</span>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+            <tr *ngFor="let permission of filteredPermissions">
+              <td class="permission">{{ permission }}</td>
+              <td *ngFor="let role of roles" class="access text-center">
+                <span title="{{ role | translate }}">
+                  <i
+                    class="fa fa-square"
+                    [ngClass]="{'text-success': permissionsByRoles[role][permission], 'text-danger': !permissionsByRoles[role][permission]}">
```

Comment:
As mentionned this morning, a third value is possible.
In such case (more than 2 choices), it's more readable if each class is on a separate line (my opinion)